core/src/main/java/org/hibernate/ogm/dialect/impl/IdentifiableDriver.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.dialect.impl;
+
+import org.hibernate.resource.transaction.TransactionCoordinator.TransactionDriver;
+
+/**
+ * @author Davide D'Alto
+ */
+public interface IdentifiableDriver extends TransactionDriver {
```

Comment:
The name and/or javadoc could use some improvement, as I only get some hint of what "IdentifiableDriver" means when looking at what it extends.